builtin/logical/database/backend_test.go

```diff
@@ -147,7 +147,7 @@ func TestBackend_RoleUpgrade(t *testing.T) {
 	}
 
 	// Upgrade case
-	badJSON := `{"statments":{"creation_statments":"test","revocation_statements":"","rollback_statements":"","renew_statements":""}}`
+	badJSON := `{"statements":{"creation_statements":"test","revocation_statements":"","rollback_statements":"","renew_statements":""}}`
```

Comment:
[For others in the Vault team] This may not be required.

Reply:
Will break things.